app/src/main/java/io/github/droidkaigi/confsched/api/DroidKaigiClient.java

```diff
@@ -28,11 +30,13 @@
 
     private static final String END_POINT = "https://raw.githubusercontent.com";
     private static final String GOOGLE_FORM_END_POINT = "https://docs.google.com/forms/d/";
+    private static final String GITHUB_END_POINT = "https://api.github.com";
```

Comment:
It's not bad for me because easy to understand.

But, I think also this is not needed to define.
```java
Retrofit githubRetrofit = new Retrofit.Builder()
                .baseUrl("https://api.github.com")
                .build();
```